amalgamation/Makefile

```diff
@@ -23,6 +23,11 @@ ifndef OPENBLAS_ROOT
     export OPENBLAS_ROOT=/usr/local/opt/openblas
 endif
 
+# use F16C if the architecture supports it, turned off by default
+ifndef USE_F16C
```

Comment:
what does 16C stand for?

Reply:
added better comment. F16C is an instruction set extension supported by newer x86 CPUs. It provides intrinsics for faster fp16 compute.